packages/components-date/src/FieldTime/FieldTime.stories.tsx

```diff
@@ -38,31 +44,51 @@ export default {
   title: 'Date / FieldTime',
 }
 
-const Template: Story<FieldTimeProps> = (args) => <FieldTime {...args} />
+const Template: Story<FieldTimeProps & { externalLabel: boolean }> = ({
+  externalLabel,
```

Comment:
Doing this would remove the need to make any changes to existing stories.

```suggestion
  externalLabel = true,
```